### DISCUSSION TITLE
Issue with APP_TITLE Customization Not Reflecting in Webpage Tab Title when reloading

### DISCUSSION THREAD

**Discussion by mkagit** [danny-avila/LibreChat#3885] [Troubleshooting]
### What happened?

It's a simple problem, but it's important for those who want or need to change the app's name.

In the .env file, there is an option for users to customize the app's name instead of using "Librechat".
However, **when reloading** the pages, the name Librechat still appears in the webpage tab title, alternating between the customized APP_TITLE and the Librechat title. On the discussions and Discord server, various suggestions have been made, such as editing the index.html and the vite.config.ts and then doing a local build, but this doesn't resolve the issue.

### Steps to Reproduce

1. Set the custom APP_TITLE variable in the .env

### What browsers are you seeing the problem on?

Chrome, Microsoft Edge, Mobile (Android)

### Screenshots

https://github.com/user-attachments/assets/8c1cb251-285d-4cdd-9974-349bc41a9d6c

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
https://github.com/danny-avila/LibreChat/discussions/2300

**Reply by danny-avila** [reply]
There are a few places to edit, not just the two mentioned in the existing discussion. Doing a case-sensitive file search for `LibreChat` will show you where.

**Reply by mkagit** [reply]
Edited every LibreChat word I could found possibly related related to the issue. Still showing LibreChat and the APP_TITLE as I show in the attached video, but now only on the login webpage title when reloading.

**Reply by danny-avila** [reply]
It could be a caching issue, restart the server, then try on a private window

**Reply by mkagit** [reply]
Danny, tried using private and cache disabled.
Did a new build and still can't fix the login webpage title bouncing between the APP_TITLE and LibreChat.
Could you please look in the code and try to point it out where exactly to change, I think more users will be interested in this feature. And if we could do it all through the env or yaml would be awesome, so no need to local build. 
In my case I need to set a local translated name to the webapp.

**Reply by harrisonhxy** [reply]
> Danny, tried using private and cache disabled. Did a new build and still can't fix the login webpage title bouncing between the APP_TITLE and LibreChat. Could you please look in the code and try to point it out where exactly to change, I think more users will be interested in this feature. And if we could do it all through the env or yaml would be awesome, so no need to local build. In my case I need to set a local translated name to the webapp.

May I ask if you have solved this problem effectively? I have not been able to resolve it so far. It's also worth stating that I deployed librechat with docker

**Reply by rm-1997** [reply]
same here - wondering whether this problem has been solved ...

**Reply by harrisonhxy** [reply]
> same here - wondering whether this problem has been solved ...

Currently, my solution is to edit the docker-compose.override.yml file using the method of building a local image. This way, when refreshing, the name of the APP_TITLE page tag will no longer appear as Librechat.

However, there is also a problem that the Librechat issue still occurs when refreshing the registration page. I'm not sure if I have missed some files that still need to be modified?

**Reply by mkagit** [reply]
To local build every time is resource consuming. It was supposed to be working from the env but it won't.

**Comment by markvshaney** [comment]
I've tried to unsubscribe but still getting spammed😥

**Reply by mkagit** [reply]
Spammed? It's a new thread
